ontologies/vocabs.ttl

```diff
@@ -1083,6 +1090,48 @@ The tool was unable to determine whether the malware binary is malicious."""^^xs
 	skos:prefLabel "unknown"^^xsd:string ;
 	.
 
+gist:_PatternType_Pcre
+	a gist:AccountType ;
+	skos:definition """STIX 2.1 description:
+Specifies the Perl Compatible Regular Expressions language [PCRE]."""^^xsd:string ;
+	skos:prefLabel "pcre"^^xsd:string ;
+	.
```

Comment:
Case inconsistency:  Pcre & pcre
Or maybe label should be words spelled out and [create and] use gist:abbreviation  for the short form (make it a subproperty of skos:note).